apollo-compiler/src/test/graphql/com/example/input_object_type/TestQueryExpected.java

```diff
@@ -0,0 +1,86 @@
+package com.example.input_object_type;
+
+import com.apollostack.api.Query;
+import java.lang.Override;
+import java.lang.String;
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+public final class TestQuery implements Query<TestQuery.Variables> {
```

Comment:
Do we need to distinguish between `GraphQLQuery` and `GraphQLMutation`. I checked iOS, `GraphQLQuery`, `GraphQLMutation` are just empty extension of `GraphQLOperation` without any additional functionality, so difference is only in naming for clarity, right?